Libplanet/Action/NullAction.cs

```diff
@@ -0,0 +1,33 @@
+using Bencodex.Types;
```

Comment:
All newly added files should turn on the null checker.  See also <https://github.com/planetarium/libplanet/issues/458#issuecomment-669674809> for details.

```suggestion
#nullable enable
using Bencodex.Types;
```